tests/remove.rs

```diff
@@ -0,0 +1,40 @@
+use test_harness::test;
+use test_harness::BaseMatcher;
+use test_harness::GhciWatch;
+
+#[test]
+async fn can_remove_multiple_modules_at_once() {
+    let mut session = GhciWatch::new("tests/data/simple")
+        .await
+        .expect("ghciwatch starts");
+    session
+        .wait_until_ready()
+        .await
+        .expect("ghciwatch loads ghci");
+
+    session.fs_mut().disable_load_bearing_sleep();
```

Comment:
This API is clumsy, the load bearing sleep value should be tied to the particular `Fs` instance, not shared.